packages/rpc/src/Base/index.ts

```diff
@@ -543,3 +543,5 @@ export class Base {
     return this.#rpcProperties;
   }
 }
+
+export type Option<T> = T | null;
```

Comment:
`Nullable` is more semantic

Reply:
`Option<T>` here is for keeping the same with the `jsonrpc-types` https://github.com/nervosnetwork/ckb/blob/0c33675e11ec072fbbe41c135a88749167f204d0/util/jsonrpc-types/src/cell.rs#L49C15-L49C21